Neos.Media.Browser/Resources/Private/Templates/Usage/RelatedNodes.html

```diff
@@ -34,26 +34,34 @@
                     </td>
                     <td>
                         <f:switch expression="{true}">
-                            <f:case value="{inaccessibleRelation.workspace.personalWorkspace}">
+                            <f:case value="{inaccessibleRelation.relevantWorkspaceMetadata.personalWorkspace}">
                                 <i class="fas fa-user"
                                    title="{neos:backend.translate(id: 'workspaces.personalWorkspace', source: 'Main', package: 'Neos.Workspace.Ui')}"
                                    data-neos-toggle="tooltip"></i>
                                 {neos:backend.translate(id: 'workspaces.personalWorkspace', source: 'Main', package: 'Neos.Workspace.Ui')}
-															  {inaccessibleRelation.label}
+                                <f:if condition="{inaccessibleRelation.relevantWorkspaceMetadata.owner}">
+                                    <f:then>
+                                        (@{inaccessibleRelation.relevantWorkspaceMetadata.owner})
```

Comment:
This leads to an error 
```suggestion
                                        (@{inaccessibleRelation.relevantWorkspaceMetadata.owner.label})
```

Reply:
Thx for testing!

This will be solved by not giving full Entity into template